src/confidential-proofs/confidential-proofs.service.ts

```diff
@@ -48,9 +48,19 @@ export class ConfidentialProofsService {
         data: serializeObject(data),
         timeout: 10000,
       })
-    );
+    ).catch(error => {
+      return error.response;
+    });
 
     if (status !== HttpStatus.OK) {
+      if (status === HttpStatus.NOT_FOUND) {
```

Comment:
I ended up just swallowing any error for reliability, but figured I would leave this since it can help improve error messages a bit and makes it easier to catch in other places